.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=VarnitaMakrariya
```

Comment:
Don't commit `env`. Add it in the gitignore

Reply:
Done